content/millennium/r4/request-and-response/communication.md

```diff
@@ -15,7 +15,7 @@ The following fields are returned if valued:
 
 * [Communication Id](https://hl7.org/FHIR/communication-definitions.html#Communication.identifier)
 * [Status (Completed)](https://hl7.org/FHIR/communication-definitions.html#Communication.status)
-* [Communication (Category Message, Reminder, Coding Query)](https://hl7.org/FHIR/communication-definitions.html#Communication.category)
+* [Category](https://hl7.org/FHIR/communication-definitions.html#Communication.category)
```

Comment:
This part is strictly dictated by the SDs. Make sure to talk with them that these labels are up to date

Reply:
Will do in meeting tomorrow